host/host_windows.go

```diff
@@ -261,3 +287,10 @@ func KernelVersionWithContext(ctx context.Context) (string, error) {
 	_, _, version, err := PlatformInformation()
 	return version, err
 }
+
+type MSAcpi_ThermalZoneTemperature struct {
```

Comment:
Don't export this type. There are similarly exported types but this is a mistake as Windows-only, while gopsutil aims to provide a platform-agnostic API.

Reply:
>Don't export this type. 

MSAcpi_ThermalZoneTemperature struct only used for WMI . I found other package use WMI also definded like it. 

>There are similarly exported types but this is a mistake as Windows-only

I don't understand what's mean similarly exported types, can you tell me detail, thanks.

Reply:
Lomanic is correct. We want to gopsutil platform independent. If this API exported only for Windows, it becomes Windows only.

Reply:
Actually, we already has exported structs on platform dependent part, but these are fault and should be fixed.

Reply:
I update `MSAcpi_ThermalZoneTemperature` to `msacpi_ThermalZoneTemperature`, but `WMI` query error, so i think this struct not modify. If you know how to change, can you tell me? I will change it. Thanks.

Reply:
`msAcpi_ThermalZoneTemperature` works properly on Windows 7 (needs admin rights though). I'm merging and fixing this unwanted export. Thanks for your contribution @pytimer.

Reply:
Thanks for your fix.